helm/hpcc/templates/_helpers.tpl

```diff
@@ -569,6 +569,7 @@ vaults:
   {{- range $vault := . }}
     - name: {{ $vault.name }}
       kind: {{ $vault.kind }}
+      namespace: {{ $vault.namespace }}
```

Comment:
Doesn't this need to be inside a check that namespace is set.

```
{{- if $value.namespace }}
```

Reply:
Yes, missed that, thanks.

Reply:
@ghalliday Updated to only add namespace if one is provided.